inc/auth.class.php

```diff
@@ -235,15 +235,24 @@ function connection_ldap($ldap_method, $login, $password) {
          if (!empty($ldap_method['sync_field'])) {
             $params['fields']['sync_field'] = $ldap_method['sync_field'];
          }
-         $infos = AuthLDAP::searchUserDn($this->ldap_connection,
-                                         ['basedn'            => $ldap_method['basedn'],
-                                               'login_field'       => $ldap_method['login_field'],
-                                               'search_parameters' => $params,
-                                               'user_params'
-                                                   => ['method' => AuthLDAP::IDENTIFIER_LOGIN,
-                                                            'value'  => $login],
-                                                   'condition'         => $ldap_method['condition'],
-                                                   'user_dn'           => $this->user_dn]);
+         try {
+            $infos = AuthLDAP::searchUserDn($this->ldap_connection, [
+               'basedn'            => $ldap_method['basedn'],
+               'login_field'       => $ldap_method['login_field'],
+               'search_parameters' => $params,
+               'user_params'       => [
+                  'method' => AuthLDAP::IDENTIFIER_LOGIN,
+                   'value'  => $login
+               ],
+               'condition'         => $ldap_method['condition'],
+               'user_dn'           => $this->user_dn
+            ]);
+         } catch (\RuntimeException $e) {
```

Comment:
To be sure to catch any execption, in case we change thrown exception on `AuthLDAP::searchUserDn()`.
```suggestion
         } catch (\Throwable $e) {
```